### ISSUE TITLE
Mozilla Firefox system menu is transparent

### ISSUE THREAD

**Issue by FredBezies** [GnuLinuxViperr/Viperr-9#1]
Simple to reproduce...

1) Install Viperr
2) Upgrade it
3) Launch Mozilla Firefox
4) Click on "hamburger menu"

![bugfx](https://cloud.githubusercontent.com/assets/8571419/17622937/ae59f6d4-609d-11e6-9031-141fa2d072fb.png)

**Comment by penthium2** [comment]
same bug for me.
Theme Dagon GTK2 issue

**Comment by FredBezies** [comment]
Isn't firefox built against gtk3 ? :)

**Comment by azgaviperr** [comment]
solution:

rm -rf $HOME/.mozilla/firefox/viperr.default/chrome
cp -r /usr/share/themes/Viperr-Dagon/Firefox/chrome/  $HOME/.mozilla/firefox/viperr.default

**Comment by penthium2** [comment]
oki je vais intégré cette modification dans viperrconf
merci azgaviperr

**Comment by penthium2** [comment]
solved with viperrconf-9.4-viperr.fc24.noarch.rpm for new user under viperr9
for old user do : 

```
rm -rf $HOME/.mozilla/firefox/viperr.default/chrome
cp -r /usr/share/themes/Viperr-Dagon/Firefox/chrome/ $HOME/.mozilla/firefox/viperr.default
```
